Sources/TensorFlow/Layers/Normalization.swift

```diff
@@ -85,12 +85,13 @@ public struct BatchNorm<Scalar: TensorFlowFloatingPoint>: Layer {
           var normalizedAxes = Array(0..<input.rank)
           normalizedAxes.remove(at: positiveAxis)
           let moments = input.moments(alongAxes: normalizedAxes)
-          runningMean.value += (moments.mean - runningMean.value) * (1 - momentum)
-          runningVariance.value += (moments.variance - runningVariance.value) * (1 - momentum)
-          let inv = rsqrt(moments.variance + epsilon) * scale
+          let decayMomentum = Tensor<Scalar>(1 - momentum)
```

Comment:
Could you please try removing the explicit `<Scalar>` generic parameter for consistency with other code?
I believe it can be inferred.
```suggestion
          let decayMomentum = Tensor(1 - momentum)
```